folium/features.py

```diff
@@ -634,6 +634,51 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
             """)  # noqa
 
 
+class Circle(Marker):
+    """Creates a Circle object for plotting on a Map.
+
+    Parameters
+    ----------
+    location: tuple or list, default None
+        Latitude and Longitude of Marker (Northing, Easting)
+    radius: int
+        The radius of the circle in meters. For setting the radius in pixel,
+        please use CircleMarker.
```

Comment:
~~please~~ use CircleMarker (no need to say please in the docs. Unless it is please read the docs :stuck_out_tongue_winking_eye:)